geometry_msgs/PoseStamped object_pose
string object_ID
---
bool finishPick